### ISSUE TITLE
Redetermination name issue

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/iRecord#439]
See forum post:
https://www.brc.ac.uk/irecord/node/7442

Is this a UKSI issue of names not matching the list being used by the scheme.

Marked as urgent as is affecting a key verifier

**Comment by johnvanbreda** [comment]
Do you know which records? When I masquerade as S. Palmer these both seem OK. Is it possible they are from a form which uses a different species list, so he needs to tick the checkbox to enable searching in UKSI rather than the form's list?

**Comment by DavidRoy** [comment]
Good point. I've asked on the forum.  Thinking about it, these might come from National Moth Night that had its own species list.

**Comment by DavidRoy** [comment]
Response from forum.  Both are NatureSpot records.

8121630 - this failed on a few attempts to redet over a couple of days but subsequently it worked. The original i/d was Metzneria lappella which was redet as Monochroa lucidella. However at the time this was resolved other Monochroa species names did not appear on the option list.

8121597 - this continued to fail despite attempts over a week or more so I decided to try a different tack. The record was sent to the recorder as a 'query' with an explanation of the problem I was having and the correct identity. I had hoped the recorder would make the correction himself so that I could accept the record but this has not happened to date. This relates to a record listed as Bryotropha domestica which was actually Exoteleia dodecella.

**Comment by johnvanbreda** [comment]
Both these records were determined on a non-UKSI taxon list, so the species Steve wanted to select were not available. I believe that if Steve had ticked the "Search all species" box the names would have been available, so I've added a rather noticeable message explaining this when a non-UKSI list was used for the original determination. Hopefully that clears the issue up.
